### ISSUE TITLE
Button "Legend" Tablet [SOLVED]

### ISSUE THREAD

**Issue by jukos** [andreaordonselli/qgis2o.gis#23]
Hello,
I can see the “Legend” button on my tablet (Android 11; Chrome browser). It is not displayed on the smartphone.
How can I hide “Legend” on my tablet?
Best regards
![legend button tablet](https://github.com/user-attachments/assets/9481a863-c466-4cf2-93b1-ad25ca6f59f8)

**Comment by andreaordonselli** [comment]
If you produced the map using the latest version of the software, the problem should not arise. Can you please clear the tablet's browser cache and reopen the map?

**Comment by jukos** [comment]
I have deleted the entire browser cache and also restarted the tablet. No change.
Of course, I can't say whether this is because the tablet is already a few years old. I don't want to rule it out. 
It is project-independent, because your demo map is also displayed with the legend button on the tablet.
Version 2.23

**Comment by andreaordonselli** [comment]
It happens when Do you start the map or after opening / closing the layerswitcher?

**Comment by andreaordonselli** [comment]
Try making this change to the `resources/o.gis/o.gis.js` file
Find this
```javascript
if (!isSmallScreen && !hasTouchScreen) {
	$('.ol-layerswitcher > button').on('click', function() {
		toggleLegendAttribution();
	});
}
```
and replace with this
```
$('.ol-layerswitcher > button').on('click', function() {
	toggleLegendAttribution();
});
```

**Comment by jukos** [comment]
Excuse me, in which line of code can I find the code? (approximately)

**Comment by jukos** [comment]
The code is not available. I suspect a problem with 2.23?
I found it in an older export test file.

**Comment by andreaordonselli** [comment]
Please, check from line 1921 to 1925 in version 2.23

**Comment by jukos** [comment]
The code is missing. No code after row 1911


![code end at row 1911](https://github.com/user-attachments/assets/c616dfb9-5bed-4e77-93de-f4aff5739c60)

**Comment by jukos** [comment]
I have now redownloaded and reinstalled the extension. The code is now there.
I will test.....

**Comment by jukos** [comment]
Works. Many thanks!

The cause must have been a faulty download of version 2.23. Sounds unlikely, but it was probably the case.

**Comment by andreaordonselli** [comment]
This issue has been resolved in the v2.24.0 release
Thanks for your report
